image/cli/mascli/functions/pipeline_config_sno

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# when SNO_MODE is detected, we have a couple of parameters that need to be set.
+function config_pipeline_sno() {
+  echo ""
+  if [[ "$SNO_MODE" == "true" ]]; then
+    # SNO Mode -- Set environment variables MONGODB_REPLICAS=1
+    echo_h2 "Info: SNO_MODE was detected."
+    export MONGODB_REPLICAS=1
+    export MONGODB_CPU_REQUESTS=300m
+    export MAS_APP_SETTINGS_AIO_FLAG=false
+    export DB2_CPU_REQUESTS=1000m ##default value is 2000m
+    export DB2_CPU_LIMITS=2000m ## default value is 4000m
+    export DB2_META_STORAGE_ACCESSMODE=ReadWriteOnce
+    export DB2_BACKUP_STORAGE_ACCESSMODE=ReadWriteOnce
+    export DB2_META_STORAGE_SIZE=10Gi
+    export DB2_BACKUP_STORAGE_SIZE=10Gi
+    export DB2_LOGS_STORAGESIZE=10Gi
+    export DB2_TEMP_STORAGE_SIZE=10Gi
+    export DB2_DATA_STORAGE_SIZE=20Gi
+    export MAS_APP_CHANNEL=8.4.x
+    export DB2_DATA_STORAGE_CLASS=gp2
+    export DB2_DATA_STORAGE_SIZE=100Gi ## default value
+    export DB2_META_STORAGE_CLASS=gp2
+    export DB2_META_STORAGE_SIZE=10Gi ## default
+    export DB2_BACKUP_STORAGE_CLASS=gp2
+    export DB2_BACKUP_STORAGE_SIZE=100Gi ## default value
+    export DB2_LOGS_STORAGE_CLASS=gp2
+    export DB2_LOGS_STORAGE_SIZE=100Gi ## default value
+    export DB2_TEMP_STORAGE_CLASS=gp2
+    export DB2_TEMP_STORAGE_SIZE=100Gi  ##default value
+    export DB2_CPU_REQUESTS=1000m ##default value
+    export DB2_CPU_LIMITS=4000m ## default value
+    export DB2_MEMORY_REQUESTS=2Gi ## default value
+    export DB2_MEMORY_LIMITS=6Gi ## default value
+    export DB2_META_STORAGE_ACCESSMODE=ReadWriteOnce
+    export DB2_BACKUP_STORAGE_ACCESSMODE=ReadWriteOnce
+    export DB2_DATA_STORAGE_ACCESSMODE=ReadWriteOnce
+    export DB2_LOGS_STORAGE_ACCESSMODE=ReadWriteOnce
+    export DB2_NAMESPACE=db2u
+    
+    prompt_for_confirm_default_yes "Do you want manage demodata to be loaded or not " MAS_APP_SETTINGS_DEMODATA && export MAS_APP_SETTINGS_DEMODATA
+  else
+    echo_h2 "Info: SNO_MODE was not detected"
+  fi
+}
+ 
```

Comment:
@sujalavelagapudi-ibm could you pls add an extra line at the end of the file to avoid lint issues